python/src/spark_rapids_ml/feature.py

```diff
@@ -386,12 +386,15 @@ def _construct_pca() -> CumlT:
 
             pca.n_cols = self.n_cols
             pca.dtype = np.dtype(self.dtype)
+            # TBD: figure out why PCA warns regardless of array order here and for singular values
             pca.components_ = cudf_to_cuml_array(
```

Comment:
Could we change it like 

``` python
order = self._fit_array_order()

np.array(self.components_, order=irder).astype(pca.dtype)
...

Reply:
This is actually a different case.  These are model params which have their own preferred order independent of the preferred order for the data.   I already set the coefficients order in pca to avoid the warning, but there seems to be a problem with 1d arrays hence the TBD.